src/H5TB-opt.c

```diff
@@ -758,15 +758,14 @@ herr_t write_records_blosc2( hid_t dataset_id,
   goto out;
  }
  int typesize = cd_values[2];
- hsize_t cshape[1];
- H5Pget_chunk(dcpl, 1, cshape);
+ hsize_t chunklen;
+ H5Pget_chunk(dcpl, 1, &chunklen);
  if (H5Pclose(dcpl) < 0)
   goto out;
- int chunklen = (int) cshape[0];
- int cstart = (int) (start / chunklen);
- int cstop = (int) (start + nrecords - 1) / chunklen + 1;
- int data_offset = 0;
- for (int ci = cstart; ci < cstop; ci ++) {
+ hsize_t cstart = start / chunklen;
+ hsize_t cstop = (start + nrecords - 1) / chunklen + 1;
+ hsize_t data_offset = 0;
```

Comment:
This one seems to be no longer used, correct?

Reply:
Yes, this can be moved downwards.  Done in 313e65f3.